README.md

```diff
@@ -30,45 +32,40 @@ Name your file with your GitHub username `username.json`; for example mine was `
   "location": ""
 }
 ```
-
-Notes on the information:
-
-> - **Name**: You can use your github name, actual name, or a pseudonym!
-
-> - **Url**: Please insert Github link here, we will expand the user json files to include socials in the future
-
-> - **Image**: You can use any _appropriate_ profile image here
-
-> - **Why** : Please write down your reason why you have joined the #FitDevs community!
-
-> - **Weekly**: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU
-
-```
-`weekly` examples :
-`["Running 6x", "Abs 6x", "Pushups 6x"]`
-`["Drinking 500ml Water 3/day", "Meatless Mondays 2x/month", "Sleeping 7h 6/week"]`
+<br>
+and fill out the information (make sure to look at the notes below)<br><br>
+
+ - Name: You can use your github name, actual name, or a pseudonym!
+ - Url: Please insert Github link here, we will expand the user json files to include socials in the future
+ - Image: You can use any appropriate profile image here
+ - Why: Please write down your reason why you have joined the #FitDevs community!
+ - Weekly: Your goals for fitness, can be food/exercise/mental/emotional, whatever fitness means to YOU
```

Comment:
I think we should bold the fields similar to how they were before. It reads a little more clear to me